From 9f2bb3db4f306d1b2255de9a3f66b9b106d5e13f Mon Sep 17 00:00:00 2001
From: Wojciech Bartczak <wbartczak@marvell.com>
Date: Wed, 16 Jun 2021 16:03:33 -0700
Subject: [PATCH 1589/1921] driver: Fixes unnecessary logging from octeontx
 info driver

This patch removes code generating noise in kernel logs.

Signed-off-by: Wojciech Bartczak <wbartczak@marvell.com>
Change-Id: I9a38904cfb856be047f9ecf3ec940b2f1e26b6f5
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/54304
Reviewed-by: Chandrakala Chavva <cchavva@marvell.com>
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/soc/marvell/octeontx_info.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/soc/marvell/octeontx_info.c b/drivers/soc/marvell/octeontx_info.c
index 309c7cf66245..bb487419c95f 100644
--- a/drivers/soc/marvell/octeontx_info.c
+++ b/drivers/soc/marvell/octeontx_info.c
@@ -196,9 +196,6 @@ static int octtx_parse_mac_info(struct device_node *node)
 			ret = of_property_read_string(node, prop_name,
 						      &board_mac);
 			if (ret) {
-				pr_info("%s: %s = %s\n",
-					 module_name(THIS_MODULE),
-					 prop_name, board_mac);
 				brd.mac_addrs[u].num = 0;
 			} else {
 				if (!kstrtou64(board_mac, 16, &mac_addr.num))
-- 
2.31.1

